### PR TITLE
Aruco/Multitag 36h11 support

### PR DESCRIPTION
**Pull request by srimanachanta**
- Aruco pipeline now infers tag width from tag family like the AprilTag pipeline
- Removes unused Aruco and 200mm AprilTag models
- `VisionEstimation.estimateCamPosePNP()` now requires a target model instead of assuming 16h5
  - Multitarget pipeline similarly infers target model of tag family now
  - `PhotonPoseEstimator` can have target model set for on-rio multitarget

### CODE REVIEW

File: photon-client/src/components/dashboard/tabs/PnPTab.vue [hunk 1]
```diff
@@ -26,11 +26,8 @@ const interactiveCols = computed(
         { name: '2020 High Goal Outer', value: TargetModel.InfiniteRechargeHighGoalOuter },
         { name: '2020 Power Cell (7in)', value: TargetModel.CircularPowerCell7in },
         { name: '2022 Cargo Ball (9.5in)', value: TargetModel.RapidReactCircularCargoBall },
-        { name: '200mm AprilTag', value: TargetModel.Apriltag_200mm },
-        { name: '6in (16h5) Aruco', value: TargetModel.Aruco6in_16h5 },
-        { name: '6in (16h5) AprilTag', value: TargetModel.Apriltag6in_16h5 },
-        { name: '6.5in (36h11) Aruco', value: TargetModel.Aruco6p5in_36h11 },
-        { name: '6.5in (36h11) AprilTag', value: TargetModel.Apriltag6p5in_36h11 }
+        { name: '6in (16h5)', value: TargetModel.k6in_16h5 },
+        { name: '6.5in (36h11)', value: TargetModel.k6p5in_36h11 }
```
Comment (amquake) [inline photon-client/src/components/dashboard/tabs/PnPTab.vue:30]: Should still be identified as 'AprilTag'

File: photon-client/src/components/dashboard/tabs/PnPTab.vue [hunk 2]
```diff
@@ -26,11 +26,8 @@ const interactiveCols = computed(
         { name: '2020 High Goal Outer', value: TargetModel.InfiniteRechargeHighGoalOuter },
         { name: '2020 Power Cell (7in)', value: TargetModel.CircularPowerCell7in },
         { name: '2022 Cargo Ball (9.5in)', value: TargetModel.RapidReactCircularCargoBall },
-        { name: '200mm AprilTag', value: TargetModel.Apriltag_200mm },
-        { name: '6in (16h5) Aruco', value: TargetModel.Aruco6in_16h5 },
-        { name: '6in (16h5) AprilTag', value: TargetModel.Apriltag6in_16h5 },
-        { name: '6.5in (36h11) Aruco', value: TargetModel.Aruco6p5in_36h11 },
-        { name: '6.5in (36h11) AprilTag', value: TargetModel.Apriltag6p5in_36h11 }
+        { name: '2023 AprilTag 6in (16h5)', value: TargetModel.kAprilTag6in_16h5 },
```
Comment (mcm001) [inline photon-client/src/components/dashboard/tabs/PnPTab.vue:29]: Do we still have the tag family drop-down in the UI that needs to be set? If so, looks like we need to remove it now?

File: photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java [hunk 3]
```diff
@@ -155,6 +157,24 @@ public void setFieldTags(AprilTagFieldLayout fieldTags) {
...
+     * <p>By default, this is {@link TargetModel#kAprilTag16h5}.
+     */
+    public TargetModel getTagModel() {
+        return tagModel;
+    }
+
+    /**
+     * Set the TargetModel representing the tags being detected. This is used for on-rio multitag.
+     *
+     * @param tagModel E.g. {@link TargetModel#kAprilTag16h5}.
```
Comment (mcm001) [inline photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java:172]: I think this link is broken? Not sure, coz it doesn't include the size like we do in TargetModel above

File: photon-client/src/components/dashboard/tabs/PnPTab.vue (hunk 2 above)
Comment (srimanachanta) [reply photon-client/src/components/dashboard/tabs/PnPTab.vue:29]: Wdym? Also, PNPTab is dead code.

Comment (mcm001) [reply photon-client/src/components/dashboard/tabs/PnPTab.vue:29]: On the apriltag tab, is there a drop-down for tag family?

Comment (srimanachanta) [reply photon-client/src/components/dashboard/tabs/PnPTab.vue:29]: Yes, does the tag family settings prop no longer exist?

File: photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java (hunk 3 above)
Comment (amquake) [reply photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java:172]: This is `photon-targeting`'s `TargetModel`, not `photon-core`'s. We should see if merging these is possible.

Comment (mcm001) [reply photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java:172]: Yeah totally. Or rename one of them lol

File: photon-client/src/types/PipelineTypes.ts [hunk 4]
```diff
@@ -225,7 +222,7 @@ export type ConfigurableAprilTagPipelineSettings = Partial<
 export const DefaultAprilTagPipelineSettings: AprilTagPipelineSettings = {
   ...DefaultPipelineSettings,
   cameraGain: 75,
-  targetModel: TargetModel.Apriltag6in_16h5,
```
Comment (srimanachanta) [inline photon-client/src/types/PipelineTypes.ts:228]: Remove k modifier

File: photon-client/src/types/PipelineTypes.ts [hunk 5]
```diff
@@ -268,7 +265,7 @@ export type ConfigurableArucoPipelineSettings = Partial<Omit<ArucoPipelineSettin
 export const DefaultArucoPipelineSettings: ArucoPipelineSettings = {
   ...DefaultPipelineSettings,
   outputShowMultipleTargets: true,
-  targetModel: TargetModel.Aruco6in_16h5,
```
Comment (srimanachanta) [inline photon-client/src/types/PipelineTypes.ts:271]: ditto

**Comment by mcm001** [comment]
Java lint, but otherwise LGTM

**Review by mcm001 — APPROVED** [review]

**Pull request merged**